packages/create-vite/template-solid-ts/pnpm-lock.yaml

```diff
@@ -0,0 +1,806 @@
+lockfileVersion: 5.3
```

Comment:
Beside the fact that I'm not sure if we want to prefer `pnpm` as default, do we need a `package-lock.json` AND a `pnpm-lock.yaml`?

Reply:
Will be better if this files are removed